docs/getting-started.md

```diff
@@ -2,12 +2,21 @@
 
 *If you've previously installed gulp globally, run `npm rm --global gulp` before following these instructions.*
```

Comment:
Should this be the place where we point to the Gulp Sip article or further below?